code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -7,14 +7,16 @@
 //HALF_LIFE(0) -> Reaction completes immediately (default chems)
 //HALF_LIFE(1) -> Half of the reagents react immediately, the rest over the following ticks.
 //HALF_LIFE(2) -> Half of the reagents are consumed after 2 chemistry ticks.
-//HALF_LIFE(3) -> Half of the reagents are consumed after 3 chemistry ticks.
+//HALF_LIFE(3) -> Half of the reagents are consumed after 3 chemistry ticks. HALF LIFE 3 CONFIRMED?

```

Comment:
Funny. Not sure where our contribution guidelines stand on it, but I'd be okay with this staying in. Might need other dev input on this, though.

Reply:
Will be removed. Just a funny thing for people to look at.

Reply:
No fun allowed.
```suggestion
//HALF_LIFE(2.2) -> Half of the reagents are consumed after 2.2 chemistry ticks.
```